dll/win32/shell32/CDefView.cpp

```diff
@@ -1343,16 +1343,29 @@ LRESULT CDefView::OnCreate(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &bHandl
     }
 
     SHChangeNotifyEntry ntreg[1];
-    ntreg[0].fRecursive = FALSE;
-    ntreg[0].pidl = m_pidlParent;
+    ZeroMemory(ntreg, sizeof(ntreg));
```

Comment:
I think you could remove this and do
`SHChangeNotifyEntry ntreg[1] = {};` above.

Also, why `ntreg[1]` ? If there's just one item, just do `SHChangeNotifyEntry ntreg = {};`
And then, the `ntreg[0]` below could then be changed to `ntreg`.

Reply:
Fixed.